guidelines/enhancement_template.md

```diff
@@ -272,6 +272,40 @@ to implement the design.  For instance,
  > 1. This requires exposing previously private resources which contain sensitive
   information.  Can we do this?
 
+## Security
+
+This section should cover the security impact of the change, if any. For example,
+does it introduce new privileges or capabilities? Does it expose new risks?
+
+### Credentials and RBAC
+
+What credentials will the new component be using if it needs to talk to either the control plane or other components? What RBAC needs to be applied?
+
+Consider the following:
+- Which service accounts will be affected by the change, what permissions will the feature code need?
+- Will end users need to make any security considerations when enabling this feature?
+- What is the scope of the required RBAC?
+  - Are cluster wider permissions required?
+  - Should RBAC be limited to only the namespace(s) where the feature is enabled?
+  - Should RBAC be limited to just the node(s) where the feature is enabled?
+  - Are we adding new permissions or escalation paths from Nodes to the Cluster level?
```

Comment:
being more specific about: can a token on node/A be used to read or write information about a different node would likely help the answers.

Reply:
Added an extra "i.e."